analyzer/tests/unit/test_option_parser.py

```diff
@@ -257,9 +254,21 @@ def test_ignore_flags_clang(self):
             'command': "clang++ {} main.cpp".format(' '.join(ignore)),
             'directory': ''}
         res = log_parser.parse_options(action)
-        ignore.remove("-E")
-        ignore.remove("-M")
-        self.assertEqual(res.analyzer_options, ignore)
+        self.assertEqual(res.analyzer_options, ["-fsyntax-only"])
+
+    @unittest.skip("This will be enabled when we distinguish -Xclang params.")
+    def test_ignore_groups(self):
```

Comment:
```suggestion
    def test_ignore_xclang_groups(self):
```

Please add a short documentation to the test.